zilliqa/src/node.rs

```diff
@@ -589,6 +590,10 @@ impl Node {
         self.consensus.get_transaction_by_hash(hash)
     }
 
+    pub fn txpool_content(&self) -> TxPoolContent {
+        self.consensus.txpool_content()
```

Comment:
FWIW I'd consider just making `transaction_pool` in `Consensus` `pub` instead and accessing it directly here. We already do similar for other fields and it saves an extra level of functions.

Reply:
Or maybe not given you have to filter out transactions with low nonces anyway.